doc/source/whatsnew/v1.3.0.rst

```diff
@@ -625,6 +625,7 @@ Other
 - Bug in :class:`Styler` where multiple elements in CSS-selectors were not correctly added to ``table_styles`` (:issue:`39942`)
 - Bug in :meth:`DataFrame.equals`, :meth:`Series.equals`, :meth:`Index.equals` with object-dtype containing ``np.datetime64("NaT")`` or ``np.timedelta64("NaT")`` (:issue:`39650`)
 - Bug in :func:`pandas.util.show_versions` where console JSON output was not proper JSON (:issue:`39701`)
+- Bug in :meth:`DataFrame.convert_dtypes` when called on an empty DataFrame (:issue:`40393`)
```

Comment:
Can you say what the bug is, e.g. "incorrectly raised" before "when called".

Reply:
No problem - I mentioned the exception as that seemed to read better to me. Hope it looks okay to you.